build/get_python_test_deps.py

```diff
@@ -76,10 +102,24 @@ def DownloadWebTestIfNecessary():
   _Download(WEBTEST_URL, webtest_file)
   _Untar(webtest_file)
 
+# Downloads the requests python module used to download other deps.
+def DownloadRequestsIfNecessary():
+  request_file = 'requestsPythonModule'
```

Comment:
pip is the standard way of download and installing python modules but I thought not everyone had pip but I know realize that it might be standard on most linux dists (and Mac it seems). Will look into this.